### PR TITLE
Stock-market website

### PR DESCRIPTION
**Pull request by Naledi-Dikgale**
### In this PR I did the following

- Initialized project with create-react-app
- Picked an  API that has a list of Stock related details to display.
- Created components for displaying the company's stock and numeric details.

- Implemented filter to help narrow down the search
- Added styling
- Added tests
- Deployed the app
- Recorded a project walkthrough video

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,143 @@
...
+
+## ❓ FAQ <a name="faq"></a>
+
+- **Is it allowed to copy the contents of this project and use it for personal use?**
+
+  - This project is free for copying and reusing in any way you like.
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE.md) licensed.
```
Comment (rustam-tolipov) [inline README.md:141]: **You did an astonishing job!** 

- [ ] However, the only issue is that your [MIT license link](https://github.com/Naledi-Dikgale/react-capstone/blob/working-tree/LICENSE.md) is broken. Kindly fix it and make sure it's showing the correct template. Thank you!

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello @Naledi-Dikgale,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
1. No Linters errors 💯
2. All requirements are complete 💯
3. Well-written code 💯 
4. All tests are passing 🥇 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Check the comments under the review._

You can also consider:
N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/pages/Home/Home.css [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+}
+
+.companyListItem {
+  min-height: 150px;
+  background-color: #d24379;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-self: center;
+}
```
Comment (mwanawabangona) [inline src/pages/Home/Home.css:47]: - [ ] `optional`
- [ ] Amazing job you have done! 👏 
- [ ] However, you missed out some little things on the design, some buttons
- [ ] please see screenshot below 🥂  see the circled area 

`your template`
image

`original template`
image

**Review by mwanawabangona — APPROVED** [review]
Hi @Naledi-Dikgale,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job implementing the requested changes!! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**